src/lib/FreeC/Backend/Agda/Converter/FuncDecl.hs

```diff
@@ -13,15 +13,14 @@ import           FreeC.Backend.Agda.Converter.Arg
 import           FreeC.Backend.Agda.Converter.Free
                                                 ( addFreeArgs )
 import           FreeC.Backend.Agda.Converter.Type
-                                                ( convertFuncType
-                                                , convertRecFuncType
-                                                )
+                                                ( convertLiftedFuncType )
 import qualified FreeC.Backend.Agda.Syntax     as Agda
 import           FreeC.Backend.Coq.Analysis.DecreasingArguments
                                                 ( identifyDecArgs )
 import           FreeC.Environment.LookupOrFail
 import           FreeC.IR.DependencyGraph
 import qualified FreeC.IR.Syntax               as IR
+import qualified FreeC.LiftedIR.Converter.Type as LIR
```

Comment:
After renaming the lifting functions, it is not necessary anymore to qualify imports of `FreeC.LiftedIR.Converter.*`.

```suggestion
import           FreeC.LiftedIR.Converter.Type
```